app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -45,13 +48,20 @@ class OverdueScreen(context: Context, attrs: AttributeSet) : RelativeLayout(cont
   }
 
   private fun setupOverdueList() {
-    overdueRecyclerView.adapter = OverdueListAdapter()
+    overdueRecyclerView.adapter = listAdapter
     overdueRecyclerView.layoutManager = LinearLayoutManager(context)
   }
 
   private fun screenCreates() = Observable.just(OverdueScreenCreated())
 
-  fun updateOverdueList() {
-
+  fun updateOverdueList(list: List<OverdueListItem>) {
+    if (list.isEmpty()) {
+      overdueRecyclerView.visibility = View.GONE
+      viewForEmptyList.visibility = View.VISIBLE
+    } else {
+      listAdapter.submitList(list)
+      overdueRecyclerView.visibility = View.VISIBLE
+      viewForEmptyList.visibility = View.GONE
+    }
```

Comment:
Looks like you have some logic in your screen. This is something that should be moved to the controller so that it's also tested :)

Reply:
`overdueRecyclerView` will be transparent when there are no items, so you can choose to not bother changing its visibility.